huaweicloud/services/as/resource_huaweicloud_as_configuration.go

```diff
@@ -340,30 +341,31 @@ func resourceASConfigurationCreate(ctx context.Context, d *schema.ResourceData,
 		InstanceConfig: instanceConfig,
 	}
 
-	logp.Printf("[DEBUG] Create AS configuration Options: %#v", createOpts)
+	log.Printf("[DEBUG] Create AS configuration Options: %#v", createOpts)
 	asConfigId, err := configurations.Create(asClient, createOpts).Extract()
 	if err != nil {
-		return diag.Errorf("Error creating ASConfiguration: %s", err)
+		return diag.Errorf("Error creating AS configuration: %s", err)
```

Comment:
ditto. And there are lots of problems like this for error returns.

Reply:
fixed